pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -21,6 +23,8 @@ type (
 		// for additional CIDR associations, e.g. to use with separate CIDR for
```

Comment:
this comment belongs to `ExtraCIDRs`... please replace it with a comment to say that it need to be deprecated.